contents/docs/deployment/scaling-posthog.md

```diff
@@ -10,6 +10,24 @@ In that example, the database grew about 200mb a day. That means Heroku's cheape
 
 However, if you do start going beyond these numbers there are things you can do to scale up.
 
+## Handling Higher Volume with Partitions
+
+Posthog utilizes partitioned tables on Postgres 12 to reorganize the tables that store events. We recommend using the partitioned setup when your deployment's queries are timing out or getting noticeably slow. The partition function below applies event/timestamp partitioning to the event table meaning the table will be partitioned per event and each event table will be partitioned per week (this is for technical clarity only as this shouldn't affect how you interact with the client). The command allows you to specify which events to explicitly create partitions for. If none are specified, the table will only be partitioned by timestamp per week. We recommend you specify a few high volume events.
```

Comment:
Might be worth splitting this paragraph.

Also could reword "this is for technical clarity..." to something like: "partitioning events doesn't change how you use PostHog. It'll only speed things up."

I'd also add something like "If any single event has > million events a week, it's worth partitioning on that event. Any events that aren't partitioned get put together in a default bucket for that week."

"If you're using posthog-js or the snippet, it's worth partitioning on `$pageview` and `$autocapture`, as they tend to be high volume events"

Reply:
done!